tests/helpers/AbstractClientForAbstractBrowser.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
+
+namespace Behat\Mink\Tests\Driver;
+
+abstract class AbstractClient extends \Symfony\Component\BrowserKit\AbstractBrowser {}
```

Comment:
Maybe we can do this (changing extends class) using `sed` replace approach I've talked about in https://github.com/minkphp/driver-testsuite/pull/38.